### PR TITLE
Feat: update all packages openseadragon version to 4

### PR DESCRIPTION
**Pull request by Tobio89**
## 📝 Description

Update all packages' openseadragon dependencies to v.4

## ✔️ PR Type

What kind of change does this PR introduce?

- [x] Package Updates

### CODE REVIEW

File: packages/renderer/package.json [hunk 1]
```diff
@@ -38,26 +38,21 @@
...
   },
   "dependencies": {
+    "openseadragon": "^4.1.1",
     "react-reconciler": "^0.29.2"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/offscreencanvas": "^2019.7.0",
-    "@types/openseadragon": "^3.0.3",
+    "@types/openseadragon": "^3.0.10",
```
Comment (minsooshin) [inline packages/renderer/package.json:47]: types version is still 3?

File: apps/viewer-demo-v2/package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
   "devDependencies": {
     "@types/lodash": "^4.14.178",
     "@types/offscreencanvas": "^2019.7.0",
+    "@types/openseadragon": "^3.0.10",
```
Comment (minsooshin) [inline apps/viewer-demo-v2/package.json:56]: types' version is still 3?

Comment (Tobio89) [reply apps/viewer-demo-v2/package.json:56]: Yes, weirdly.
I haven't seen any issues where things aren't typed properly yet.

Comment (minsooshin) [reply apps/viewer-demo-v2/package.json:56]: got it.

**Review by minsooshin — APPROVED** [review]

**Pull request merged**